Dockerfiles/agent/Dockerfile

```diff
@@ -34,10 +34,8 @@ RUN dpkg -x /datadog-agent*_amd64.deb . \
 # Configuration:
 #   - empty config file to remove logline
 #   - enable docker check by default
-#   - enable process agent
 COPY datadog*.yaml etc/datadog-agent/
-RUN mv etc/datadog-agent/conf.d/docker.d/conf.yaml.example etc/datadog-agent/conf.d/docker.d/conf.yaml.default \
- && sed -i "s/enabled: false/enabled: true/" etc/datadog-agent/conf.d/process_agent.yaml.default
+RUN mv etc/datadog-agent/conf.d/docker.d/conf.yaml.example etc/datadog-agent/conf.d/docker.d/conf.yaml.default
```

Comment:
this used to set `enabled: true` in the legacy config file. Is the new default behavior in the containerized agent to leave the setting to `false`?

Reply:
If they read `DD_PROCESS_AGENT_ENABLED` through the `pkg/config` packet, we can just add this to YES by default in the dockerfile's env

Reply:
I'm actually not sure the history here but process collection isn't default anywhere in the old Agent so I don't think we need that here either. It will still start by default but it will only collect containers (this is the "false" case in agent 5 and 6)

Reply:
Okay, makes sense! I though this variable also disabled container collection.

LGTM then!